app/controllers/bookings_controller.rb

```diff
@@ -11,8 +12,16 @@ def create
       render :new, status: 422
     end
   end
-  
+
   def index
     @bookings = Booking.all
   end
+
+  def my_bookings
+    @bookings = Booking.all
```

Comment:
should only return bookings of current user

Reply:
could have @bookings_as_rentee and @bookings_as_rentor (two instance variables), that we'll use in two different sections in the view